### ISSUE TITLE
Inlude localization Spanish (Spain)

### ISSUE THREAD

**Issue by C4BR3R4** [Keksuccino/FancyMenu#730]
Include the language file in Spanish (Spain): es_es.json

Change log (spanish): http://c4br3r4.es/index.php?topic=1014.0
Repository: https://github.com/C4BR3R4/Traducciones_Minecraft/blob/main/Mods/assets/fancymenu/lang/es_es.json

[es_es.json](https://github.com/Keksuccino/FancyMenu/files/14545835/es_es.json)

**Comment by Keksuccino** [comment]
Well thank you very much for the translation, but there's lots of ASCII stuff (emojis) in it.
I am honest with you. I would like to include it, but not with the emojis.

**Comment by Keksuccino** [comment]
Will close this for now. Please open a new issue with the fixed version if you decide to fix it! Thank you very much!

**Comment by Keksuccino** [comment]
Btw: FancyMenu has a [Crowdin page](https://crowdin.com/project/fancymenu) and Spanish is already a listed language for FM, so if you want a fancy editor for translating the file, think about switching to Crowdin :D

**Comment by C4BR3R4** [comment]
> Well thank you very much for the translation, but there's lots of ASCII stuff (emojis) in it. I am honest with you. I would like to include it, but not with the emojis.

I added the symbols in v2 and I understand that in the official language file of v3 I should not include them.
I have already removed the ASCII symbols.

[es_es.json](https://github.com/Keksuccino/FancyMenu/files/14546121/es_es.json)

PS. It took me a long time to convert from .lang to .json and translate the new strings. To help me I had to sort the alphabet lines.
